tap/passive_tapper.go

```diff
@@ -387,10 +370,9 @@ func startPassiveTapper(harWriter *HarWriter, outboundLinkWriter *OutboundLinkWr
 			memStats := runtime.MemStats{}
 			runtime.ReadMemStats(&memStats)
 			log.Printf(
-				"mem: %d, goroutines: %d, unmatched messages: %d",
+				"mem: %d, goroutines: %d, unmatched messages:",
```

Comment:
"unmatched messages:" is not needed here

Reply:
@gadotroee fixed with https://github.com/up9inc/mizu/commit/cc5620e36795f11f41cee4d3b7622f45e9393df3